### PR TITLE
Adds new plugin, http_stats

### PR DESCRIPTION
**Pull request by randall**
This is an origin intercept-based remap plugin for exposing ATS's stats over HTTP.

### CODE REVIEW

**Comment by randall** [comment]
This is a copy/paste/merge of `stats_over_http` + `statichit` plugins with the features exposed in `stats_over_http` removed since they're covered by other plugins (`compress`) or can use features in remap (eg ACLs)

**Comment by randall** [comment]
[approve ci fedora]

File: plugins/experimental/http_stats/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+This is a remap-based origin-intercept plugin that exposes the internal Traffic Server stats. This was based on the stats_over_http plugin, but removes all the features that are natively available in remap (ACLs) or other remap-plugins (eg compression).
```
Comment (ywkaras) [inline plugins/experimental/http_stats/README.md:1]: I thought we were deprecating READMEs, documenting in sphinx.

Comment (zwoop) [reply plugins/experimental/http_stats/README.md:1]: Yeh, needs a doc page I think (not sure how strict we are on those rules though). TPTBC can decide :).

Comment (ezelkow1) [reply plugins/experimental/http_stats/README.md:1]: Im +1 on a docs page, its the first place both our SREs and I go for info on how to use a plugin

**Review by zwoop — COMMENTED** [review]
Looks good to me, but would like @ezelkow1 to review before we commit.

File: plugins/experimental/http_stats/README.md (hunk 1 above)
Comment (randall) [reply plugins/experimental/http_stats/README.md]: Docs added. readme removed.

**Comment by ywkaras** [comment]
I've gotten pulled off onto a Yahoo internal issue.  I'll try to review this before the end of the week, or someone else can step in if they like.

**Review by ezelkow1 — APPROVED** [review]
Looks fine to me but haven't had a chance to play with it yet. It is experimental anyway so not much harm if it still has issues

**Comment by zwoop** [comment]
Cherry-picked to v9.2.x

**Pull request merged**